docker/docker-compose.dev.yml

```diff
@@ -6,7 +6,7 @@ services:
       context: ./${PATH_CORE}/..
       dockerfile: ./docker/core/Dockerfile
     image: aegee/core:dev
-    command: sh -c "sh /usr/app/scripts/bootstrap.sh && nodemon -L -e 'js,json' lib/run.js | bunyan --color"
+    command: "nodemon -L -e 'js,json' lib/run.js | bunyan --color"
```

Comment:
Why did you remove the part about bootstrap.sh from here? Is it because of your addition in the bootstrap file or because it is redundant with the Dockerfile?

Reply:
it is already in the ENTRYPOINT, so we only change the CMD (the trick I mentioned above). Good explanation: http://www.johnzaccone.io/entrypoint-vs-cmd-back-to-basics/